scripts/conversion_script.py

```diff
@@ -450,6 +474,12 @@ def main():
         output.message, output.alert = generate_report_message(
             highest_level, gpg_key_file
         )
+
+        if "successfully" in output.message:
+            # NOTE: When c2r statistics on insights are not reliant on rpm being installed
+            # remove this condition (=decide only based on install_convert2rhel() result)
+            c2r_installed_undo = False
```

Comment:
This is the exception for successful conversion right now, but @r0x0d please check it because I am not sure if this won't be changed because of the report_json and whole inhibited vs failed difference.. but I am kinda expecting that the message still should be the place where I can tell if it was successfully converted.

Reply:
I think we are safe if we remove the package even if it failed to execute. As far as I remember, we won't lose the report or any other file we place on the system (with exception the ones we generate during rpm install). 

Since we try to install/update the package every time the script run, I would vote for just removing it at the end of the script in the `cleanup` function.

Reply:
even the updated pkg if it was on system before?